api.planx.uk/server.ts

```diff
@@ -251,6 +253,28 @@ passport.deserializeUser(function (obj: Express.User, cb) {
 
 const app = express();
 
+// Swagger documentation config
+const options = {
```

Comment:
The `failOnErrors: true` option looks helpful and will probably help identify issues https://www.npmjs.com/package/swagger-jsdoc

Reply:
Added!